apertium-crh.crh.lexc

```diff
@@ -8,65 +9,82 @@
 !!!                                                                         !!!
 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 
+! See List of symbols http://wiki.apertium.org/wiki/List_of_symbols
+
+! See http://wiki.apertium.org/wiki/Crimean_Tatar and http://wiki.apertium.org/wiki/Category:Turkic_languages
+
 Multichar_Symbols 
 
-%<n%>             ! Noun                        !
-%<np%>            ! Proper noun                 !
-%<adj%>           ! Adjective                   !
-%<prn%>           ! Pronoun                     !
-%<post%>          ! Postposition                !
-%<adv%>           ! Adverb                      !
-%<v%>             ! Verb                        !
-%<cnjcoo%>        ! Coordinating conjunction    !
-%<cnjsub%>        ! "subordinating conjunction" (ki)
-%<cnjadv%>        ! 
+%<n%>             ! Noun                        
+%<np%>            ! Proper noun                 
+%<adj%>           ! Adjective                   
+%<prn%>           ! Pronoun                     
+%<post%>          ! Postposition                
+%<adv%>           ! Adverb                      
+%<v%>             ! Verb                        
+%<cnjcoo%>        ! Coordinating conjunction    
+%<cnjsub%>        ! Sub-ordinating conjunction
```

Comment:
Fix, and restore additional comment

Reply:
What is the problem you are referring to?

Reply:
Yeah done